dev/com.ibm.ws.kernel.service/bnd.bnd

```diff
@@ -29,7 +29,10 @@ Export-Package: \
  
 Private-Package: \
  com.ibm.ws.kernel.server.internal, \
- com.ibm.ws.kernel.service.utils.resources
+ com.ibm.ws.kernel.service.utils.resources, \
+ !com.ibm.lang.management
```

Comment:
Is this supposed to be com.ibm.lang.management?  I thought by default it would not be put in there.  If that is the case what value is there in specifying it in the Private-Package?

Reply:
I put this in to prevent the jar packaging putting that class in the jar. I created a stub class to compile against because it was simpler than trying to make the IBM Java interfaces available to compile against, however we don't want to actually use it.